index.html

```diff
@@ -4922,7 +4922,7 @@ <h3>Base Context</h3>
           "@type": "@id"
         },
         "name": {
-          "@id": "https://schema.org/description",
+          "@id": "https://schema.org/name",
           "@type": "http://www.w3.org/2001/XMLSchema#string"
         },
```

Comment:
It seems this should just be:

"name": "https://schema.org/name",

Reply:
Yes, agree (and change applied).